sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/DocumentProducer.java

```diff
@@ -71,15 +73,15 @@ void populatePartitionedQueryMetrics() {
                                                              QueryMetricsConstants.RequestCharge,
                                                              pageResult.getRequestCharge());
                 ImmutablePair<String, SchedulingTimeSpan> schedulingTimeSpanMap =
-                        new ImmutablePair<>(targetRange.getId(), fetchSchedulingMetrics.getElapsedTime());
+                        new ImmutablePair<>("999", fetchSchedulingMetrics.getElapsedTime());
```

Comment:
999?
leftover code for testing?

Reply:
Yeah. Need to delete some of the old partitionrange related stuff. For now just added some place holders. Need to see what to do with querymetrics.